src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringTest {
+
+    @Test
+    void srs1Test() {
+        String value1 = "1,2";
+        assertThat(value1.split(",")).contains("1");
+        assertThat(value1.split(",")).contains("2");
+        assertThat(value1.split(",")).containsExactly("1", "2");
+
+        String value2 = "1";
+        assertThat(value2.split(",")).contains("1");
+        assertThat(value2.split(",")).containsExactly("1");
```

Comment:
```suggestion
        assertThat("1".split(",")).containsExactly("1");
```

로컬 변수를 제거하면 value2와 같이 로컬 변수 이름을 짓는데 낭비하는 시간도 줄일 수 있지 않을까?